huaweicloud/services/ecs/resource_huaweicloud_compute_instance.go

```diff
@@ -410,6 +410,13 @@ func ResourceComputeInstance() *schema.Resource {
 							ForceNew:     true,
 							RequiredWith: []string{"bandwidth.0.size"},
 						},
+						"extend_param": {
+							Type:         schema.TypeMap,
+							Optional:     true,
+							ForceNew:     true,
+							Elem:         &schema.Schema{Type: schema.TypeString},
+							RequiredWith: []string{"bandwidth.0.charge_mode"},
+						},
```

Comment:
we can remove the `RequiredWith`

```suggestion
						"extend_param": {
							Type:     schema.TypeMap,
							Optional: true,
							ForceNew: true,
							Elem:     &schema.Schema{Type: schema.TypeString},
						},
```